var/spack/repos/builtin/packages/cmake/package.py

```diff
@@ -132,6 +132,11 @@ class Cmake(Package):
     # https://gitlab.kitware.com/cmake/cmake/issues/18232
     patch('nag-response-files.patch', when='@3.7:3.12')
 
+    # Cray libhugetlbfs and icpc warnings failing CXX tests
+    # https://gitlab.kitware.com/cmake/cmake/-/merge_requests/4698
+    # https://gitlab.kitware.com/cmake/cmake/-/merge_requests/4681
+    patch('ignore_crayxc_warnings.patch', when='@3.7:3.17')
```

Comment:
If 3.17.2 is the last version that suffers from this problem, we should specify that here.

Reply:
@becker33  Fixed.